dependency-suppressions.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Can we put this file into its own directory under the root `config` directory?

Reply:
Sure https://github.com/openbrewerydb/openbrewerydb-java-api/pull/20/commits/55aaaf75a34d94f523973425b90a7420f0c6c70e